tests/foreman/cli/test_host.py

```diff
@@ -665,7 +665,7 @@ def test_add_parameter_diff_values(self):
                 self.assertEqual(value, self.host['parameters'][name])
 
     @tier1
-    def test_add_parameter_by_host_name(self):
+    def test_positive_add_parameter_by_host_name(self):
```

Comment:
from this test and below, I am specifically using `host_name` and `host_id` instead of `name` and `id` to differentiate it clearly with the parameter name/value.